src/ContentValidationListener.php

```diff
@@ -225,6 +255,20 @@ public function onRoute(MvcEvent $e)
     }
 
     /**
+     * @param string $controllerService
+     *
+     * @return boolean
+     */
+    protected function isAllowsOnlyFieldsInFilter($controllerService)
```

Comment:
I'm going to remove the `is` prefix here to make it easier to understand. (The method is checking if it _allows only fields in the filter_.)